specs/spec_windows.go

```diff
@@ -1,5 +1,13 @@
 package specs
 
+// Temporary Windows version of the spec in lieu of opencontainers/specs having
+// Windows support currently.
+
+type (
+	PlatformSpec WindowsSpec
+	ProcessSpec  Process
```

Comment:
Missing `Spec ocs.Spec` here, no?

Actually, this one should be generic and could be in `specs/spec.go` directly.